### DISCUSSION TITLE
Use the latest TRegExpr which supports 'lookahead assertions'

### DISCUSSION THREAD

**Discussion by Alexey-T** [doublecmd/doublecmd#1339] [Suggestions]
Current DC 1.1.2~71 has old TRegExpr from old FPC which don't support SUBJ.

![Screenshot from 2023-11-09 18-31-21](https://github.com/doublecmd/doublecmd/assets/4154978/61e1491a-b3fc-4544-808e-1ff9d466cf3a)

![Screenshot from 2023-11-09 18-31-23](https://github.com/doublecmd/doublecmd/assets/4154978/bb9016e5-a0ce-4e61-b6e5-a51c2b6e794d)

**Comment by alexx2000** [comment]
DC use actual TRegExpr where is possible. But search in the SynEdit is a part of Lazarus. So I am nothing can do in this case.
